src/components/Deck/Deck.jsx

```diff
@@ -1,11 +1,16 @@
-// import Card from "@/components/Card/Card";
-// import "./Deck.scss";
-// import cardsData from "@/data/fakeCards.json";
+import React from "react";
+import Card from "@/components/Card/Card";
+import "./Deck.scss";
+import cardsData from "@/data/fakeCards.json";
 
-// const Deck = () => {
-// you will need to map over the cardsData array and render a Card component for each card object
-// how will you pass props to the Card component?
-
-// };
+const Deck = () => {
+    return <div className="deck">
+        {cardsData.map((card) => (
+            <Card key={card.id} title={card.title} link={card.url} image={card.image} />
+        ))}
+    </div>
+};
 
```

Comment:
If you consider more clear name would better "deck" like "cardDeck".

Reply:
ohh okay so like cardsdata.map((cardDeck) => ..... 
and then for the key= [cardDeck.id} title={cardDeck.title} and so on? 
Is that what you mean?

Reply:
I asume she meant const CardDeck = () => {...
because that is a card deck, within which you are mapping each individual card